### DISCUSSION TITLE
Does this driver support DLSS and Reflex?

### DISCUSSION THREAD

**Discussion by shichang00** [NVIDIA/open-gpu-kernel-modules#527] [Q&A]
As the title，i want to use DLSS, but it cant be open

**Comment by memeruiz** [comment]
DLSS and Raytracing works. I think they depend more on libraries (closed sourced) than the kernel driver. Cyberpunk 2077 is detecting it. I'm using wine-GE

Reflex, not sure. I think most of these features are related more to the closed-sourced libraries.

You need these libraries for DLSS and Raytracing installed:
libnvidia-ngx1 libnvoptix1 libnvidia-rtcore
